src/mdyml/convert_cisagov.py

```diff
@@ -0,0 +1,211 @@
+            possible_header_columns = line.rstrip().strip("|").split("|")
+            if len(possible_header_columns) == EXPECTED_COLUMN_COUNT:
+                in_table = True
+                skip_next_line = True
+            continue
+        if in_table and not line.startswith("|"):
+            logging.warning("Line does not start with |, skipping: %s", line)
+            continue
+        if in_table:
+            logging.debug("Appending row")
+            table_rows.append(line)
+    logging.info("Done reading markdown table %d rows.", len(table_rows))
+
+    # Process all the data into a list of dictionaries
+    out_dict_list = []
+    row_count = 0
+    for row in table_rows:
+        row_count += 1
+        in_row_list = row.split("|")[1:-1]
+        logging.debug(
+            "Processing line %d with %d columns: %s", row_count, len(in_row_list), row
+        )
+        if len(in_row_list) != EXPECTED_COLUMN_COUNT:
+            logging.warning(
+                "Skipping line %d with unexpected number of columns %d: %s",
+                row_count,
+                len(in_row_list),
+                in_row_list,
+            )
+            continue
+        # Trim whitespace from each field
+        in_row_list = [field.strip() for field in in_row_list]
+        # Create a dictionary from the row data
+        in_row_dict = dict(zip(EXPECTED_COLUMN_NAMES, in_row_list))
+        # Create a dictionary for our calculated output
+        out_dict = {}
+        out_dict["vendor"] = in_row_dict["vendor"]
+        out_dict["product"] = in_row_dict["product"]
+        out_dict["cves"] = {}
+        for cve in ORDERED_CVE_IDS:
+            out_dict["cves"][cve] = {
+                "investigated": False,
+                "affected_versions": [],
+                "fixed_versions": [],
+                "unaffected_versions": [],
+            }
+
+        # Determine where the data from the versions column is routed using the status column
+        if in_row_dict["versions"]:
+            # Spit versions out by comma and trim whitespace
+            versions = [v.strip() for v in in_row_dict["versions"].split(",")]
+            match in_row_dict["status"].lower():
+                case "not affected":
+                    out_dict["cves"][DEFAULT_CVE_ID]["investigated"] = True
+                    out_dict["cves"][DEFAULT_CVE_ID]["unaffected_versions"] = versions
+                case "affected":
+                    out_dict["cves"][DEFAULT_CVE_ID]["investigated"] = True
+                    out_dict["cves"][DEFAULT_CVE_ID]["affected_versions"] = versions
+                case "fixed":
+                    out_dict["cves"][DEFAULT_CVE_ID]["investigated"] = True
+                    out_dict["cves"][DEFAULT_CVE_ID]["fixed_versions"] = versions
+                case _:  # anything else; unknown, or missing
+                    out_dict["cves"][DEFAULT_CVE_ID]["affected_versions"] = versions
+
+        # Extract link from markdown
```

Comment:
More capitalization:
```suggestion
        # Extract link from Markdown
```